### PR TITLE
installer: copy rnnoise.dll

### LINKED ISSUE #3477: Installer doesn't copy rnnoise.dll
I am trying to build the mumble Installer but am having trouble getting the installer to properly work...
using Instructions here https://wiki.mumble.info/wiki/BuildingWindows
My locally compiled version of Mumble builds and works perfectly fine 

The installer compiles fine, and installs fine but I must have misconfigured the environmental variables 

I receive the following error once I try to run Mumble after installing it 
":x: Failed to load mumble_app.dll " in a windows popup 

According to documentation ... latest WIX stable Version (currently 3.8)  --> in reality its currently 3.11.1 ( ***This the one I installed)

I then set some environmental variables specifically 

MumbleSourceDir =  C:\MumbleBuild\win64-static-1.3.x-2018-07-01-x\mumble
MumbleQtDir = C:\MumbleBuild\win64-static-1.3.x-2018-07-01-x\Qt5
MumbleSndFileDir = C:\MumbleBuild\win64-static-1.3.x-2018-07-01-x\sndfile
MumbleMySQLDir =C:\MumbleBuild\win64-static-1.3.x-2018-07-01-x\mariadbclient" 

MumbleNoSSE2 -- removed in the config because it was giving issues with build 

Are these build instructions up to date to build the installer? 
Are there updated instructions floating around somewhere?

Thanks :palm_tree:

### PR DESCRIPTION
**Pull request by edm7707**
Fixes #3477.

The file copy can be disabled with the `MumbleNoRNNoise` environment variable.

### CODE REVIEW

File: installer/Product.wxs [hunk 1]
```diff
@@ -131,6 +131,10 @@
       <?ifdef G15 ?>
         <ComponentRef Id="mumble_g15_helper.exe" />
       <?endif ?>
+      <?ifdef RRN ?>
+        <ComponentRef Id="rnnoise.dll" />
+      <?endif ?>
+
```
Comment (davidebeatrici) [inline installer/Product.wxs]: Extra new line.

File: installer/Files.wxs [hunk 2]
```diff
@@ -66,6 +66,12 @@
         </Component>
       <?endif ?>
 
+      <?ifdef RRN ?>
```
Comment (davidebeatrici) [inline installer/Files.wxs]: It should be `RNN`.

**Comment by davidebeatrici** [comment]
Thank you very much for your contribution!

I see this is your first pull request, you did well aside from two things:
- You used the `master` branch, while you should create a new branch for each change you propose instead. That way, you can open multiple pull requests at the same time and keep track of the changes you proposed easily.
- Your commit email (`a@b`) differs from your GitHub's profile one, meaning that the commit is not "tied" to it. You should set up Git on your system so that it uses the same email you use on GitHub. I would also set the same username.
https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup

**Comment by davidebeatrici** [comment]
Now your email and username are `=`.

How did you try to set them?

**Comment by edm7707** [comment]
When setting username I accidentally added a "=" :expressionless: ... I think I properly set the configs for the commit now

**Comment by davidebeatrici** [comment]
Superseded by #3481.

**Pull request closed without merging**